apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -422,7 +422,7 @@ export class ModalPage {
   }
 
   async switchNetworkWithNetworkButton(networkName: string) {
-    const networkButton = this.page.getByTestId('w3m-network-button')
+    const networkButton = this.page.getByTestId('wui-network-button')
```

Comment:
Is this ok ? Shouldnt it be appkit?

Reply:
Yeap, I think we don't need to pass data test id to `w3m-network-button`, the `wui-network-button` already have inside, so this is safe to do